tools/exporters/data_dictionary_builder.php

```diff
@@ -240,34 +247,33 @@
     if (empty($table)) {
         continue;
     }
-    print "Inserting validity for $table\n";
-    // Insert validity
+
+    // INSTRUMENT VALIDITY
+    print "\tInserting validity for $table\n";
     $Name = $table . "_Validity";
 
-    $_type_enum = 'enum(\'Questionable\', \'Invalid\', \'Valid\')';
```

Comment:
🤔

Reply:
@Jkat can you clarify your comment?

Reply:
I think I was very confused why this line was done this way instead of how it has been modified now on line 255: https://github.com/aces/Loris/pull/4140/files#diff-acb7e0105c9e180f46efe44050bf0e4dR255